File: Tarea-Gestor-Go/comandos/agregar.go
package comandos

import (
	"fmt"
	"gestor_tareas/manejador"
)

func AgregarTarea(man *manejador.ManejadorTareas, descripcion string) {
	if descripcion == "" {
		fmt.Println("Error: La descripción de la tarea no puede estar vacía.")
		return
	}
	man.Agregar(descripcion)
	if err := man.Guardar(); err != nil {
		fmt.Printf("Error al guardar la tarea: %v\n", err)
		return
	}
	fmt.Printf(" Tarea agregada: \"%s\"\n", descripcion)
}

File: GestorTareas/src/comandos/agregar.rs
use crate::gestor::Gestor;

pub fn ejecutar(gestor: &mut Gestor, descripcion: String) {
    match gestor.agregar(descripcion) {
        Ok(tarea_agregada) => {
            println!("Tarea agregada:");
            println!("   ID: {}", tarea_agregada.id);
            println!("   Descripción: {}", tarea_agregada.descripcion);
        }
        Err(e) => eprintln!("{}", e),
    }
}